cmd/kusk/cmd/generate.go

```diff
@@ -299,3 +239,56 @@ func init() {
 
 	apiTemplate = template.Must(template.New("api").Parse(templates.APITemplate))
 }
+
+var generateDescription = `Description:
```

Comment:
Could you also add the simplest command possible, so user are not scared with all the different options? 

```sh 
kusk generate -i spec.yaml
```